CHANGELOG.next.asciidoc

```diff
@@ -318,6 +318,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add Filebeat Okta module. {pull}16362[16362]
 - Add source field in k8s events {pull}17209[17209]
 - Improve AWS cloudtrail field mappings {issue}16086[16086] {issue}16110[16110] {pull}17155[17155]
+- Added new module `crowdstrike` for ingesting Crowdstrike Falcon streaming API endpoint event data. {pull}16988[16988]
+- Added documentation for running Filebeat in Cloud Foundry. {pull}17275[17275]
```

Comment:
just want the Changelog line for your PR.  (this happens with backports)

Reply:
Ah I missed that. Thank you!